src/InstallationValidator.Core/Services/ValidationReportingTask.cs

```diff
@@ -64,7 +64,8 @@ private ReportConfiguration createReportConfiguration(string reportSubtitle, str
             Template = _reportTemplateRepository.All().FirstOrDefault(),
             Title = _applicationConfiguration.OSPSuiteNameWithVersion,
             SubTitle = reportSubtitle,
-            ReportFile = reportOutputPath(outputFilePath, reportDateAndTime)
+            ReportFile = reportOutputPath(outputFilePath, reportDateAndTime),
+            NumberOfCompilations = 1
```

Comment:
This might help to reduce overall time. I know this was required for complex reports with cross references to ensure that toc etc... was uptodate. But we are not using any of that in this report so 1 might be enough. I tried with this settings on a few different reports and could not see any issue